docs/video/obs/quicksync.md

```diff
@@ -3,4 +3,81 @@ description: OBS configuration for Intel GPUs
 icon: simple/intel
 ---
 
-# :simple-intel: QuickSync OBS Configuration
\ No newline at end of file
+<div class="annotate" markdown>
+
+# :simple-intel: QuickSync OBS Configuration
+
+Before following this, make sure that you've followed the [initial configuration](index.md) first. **Make sure that you test your settings!** See [Replay Buffer](index.md#replay-buffer).
+
+## :material-cog: Configuration
+:material-information-slab-circle: These settings are intended for recording Minecraft at 30-60 frames per second with acceptable quality and medium file sizes. If you're looking to record with high fps, look to [NVENC](#index.md#nvenc) as an alternative to QuickSync
+
+=== ":material-speedometer: Maximum Performance"
+
+    ??? image "Recording Settings"
+        <center>
+            ![Maximum performance recording settings](../../assets/images/video/obs/quicksync/recordingsettings.png){ width="600" }
+        </center>
+
+    ??? image "Encoder Settings"
+        <center>
+            ![Maximum performance encoder settings](../../assets/images/video/obs/quicksync/encodersettings.png){ width="600" }
+        </center>
+
+    <h3>Recording Settings</h3>
+
+    - **Video Encoder:** Use **Quicksync H.264**. 
+
+    <h3>Encoder Settings</h3>
+
+    - **Target Usage** There is no visual difference between speed, balanced and quality as far as Shanki can tell.
```

Comment:
people don't know who Shanki is lol, this should have a recommendation regardless. make it more concise

Reply:
I'm crying.